clients/bazel-module-registry/src/main/kotlin/CompositeBazelModuleRegistryService.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.bazelmoduleregistry
+
+import java.io.File
+
+import org.apache.logging.log4j.kotlin.logger
+
+/**
+ * A composite Bazel module registry service that aggregates multiple [BazelModuleRegistryService] instances and
+ * delegates the package lookup to the corresponding registry.
+ * @param packagesPerRegistry A map of [BazelModuleRegistryService] instances to the list of packages they are
```

Comment:
Please use exclusively KDoc [inline markup](https://kotlinlang.org/docs/kotlin-doc.html#inline-markup).

Reply:
Please also describe how the lookup is performed in terms of priority: I guess the first match wins?